releases/8.0.0.md

```diff
@@ -71,12 +71,15 @@ Python:
 
 JavaScript:
     Upgraded:
-        primevue == 4.1.0
+        primevue == 4.2.5
```

Comment:
I couldn't bump to 4.3.x without breaking the controlled list manager. I'm going to look into it a little bit and see if I need to report an issue.